modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -200,5 +200,40 @@ controller-0$ sudo docker restart octavia_worker
 ====
 Depending on your OpenStack environment, Octavia might not support UDP
 listeners, which means there is no support for UDP Services if Kuryr SDN is
-used.
+used. Octavia support for UDP started on {rh-openstack} version 16.
 ====
+
+
+== OVN Octavia Driver
+
+Octavia supports enabling multiple provider drivers via the Octavia API.
+Since {rh-openstack} version 16, the OVN octavia provider is supported for
+OpenShift on OpenStack. Users can see all available Octavia provider drivers
+with:
++
+----
+$ openstack loadbalancer provider list
++---------+-------------------------------------------------+
+| name    | description                                     |
++---------+-------------------------------------------------+
+| amphora | The Octavia Amphora driver.                     |
+| octavia | Deprecated alias of the Octavia Amphora driver. |
+| ovn     | Octavia OVN driver.                             |
++---------+-------------------------------------------------+
+----
+
+The OVN Octavia provider driver is an integration driver between load balancer
+provided by OVN and Octavia. It supports basic load balancer functionalities
+and is based on Openflow rules. This provider driver is automatically enabled
+in Octavia by Director on OVN Neutron ML2 enabled deployments. There are no
+additional installation or configuration steps required. The Amphora provider
+driver remains enabled and as the default provider driver.
```

Comment:
Isn't the ovn-octavia provider considered the default when enabled on osp16?

Reply:
No, it will be the default for Kuryr. But in Octavia, the default will still be amphora driver. So, if you create a loadbalancer without specifying a provider, it will be an amphora load balancer